README.rst

```diff
@@ -438,10 +440,10 @@ Driver specific installation instructions can be found on Read The Docs:
 * `nidcpower <http://nimi-python.readthedocs.io/en/master/nidcpower.html#installation>`_
 * `nidmm <http://nimi-python.readthedocs.io/en/master/nidmm.html#installation>`_
```

Comment:
no NI-Digital link. Is that intentional?

Reply:
Yes, Digital is not on ReadTheDocs yet.